src/extensions/paramHelper.ts

```diff
@@ -16,6 +16,41 @@ import * as askUserForParam from "./askUserForParam";
 import * as track from "../track";
 import * as env from "../functions/env";
 
+/**
+ * Interface for holding different param values for different environments/configs.
+ */
+export interface ParamBindingOptions {
+  default: string;
+  local?: string;
+  // Add project specific key:value here when we want to support that.
+}
+
+export function getDefaultParamBindings(params: { [key: string]: ParamBindingOptions }): {
```

Comment:
Same comment as above - this sounds like it returns the default param values defined in extension.yaml